docs/docs/fundamentals/web-support.md

```diff
@@ -86,3 +86,60 @@ module.exports = {
   },
 };
 ```
+
+## Web without a Babel plugin
+
+As of Reanimated `2.15`, the Babel plugin (`react-native-reanimated/plugin`) is optional on Web.
```

Comment:
I think it is worth noting here that adding dep array doesn't break behavior on mobile, hence can be used on code that is planned to be reused across mobile and web